CHANGELOG.md

```diff
@@ -37,9 +43,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Vi cursor blinking not reset when navigating in search
 - Scrolling and middle-clicking modifying the primary selection
 - Bottom gap for certain builtin box drawing characters
-- Incorrect built-in glyphs for `U+2567` and `U+2568`
 - Character mappings in the DEC special graphics character set (line drawing)
 - Window flickering on resize on Wayland
+- Incorrect built-in glyphs for `U+2567` and `U+2568`
+- Cursor not hiding on GNOME Wayland
+- Font having different scale factor after monitor powering off/on on X11
+- Wrong mouse input when opening a new tabbed window on macOS
```

Comment:
Rather than describing the mouse part here, I'd instead mention that the entire window content was offset. It affected rendering too, even with no mouse usage.